repo/history.go

```diff
@@ -132,6 +131,6 @@ func (h *history) getCurrent(buf *bytes.Buffer) (err error) {
 		return err
 	}
 	defer f.Close()
-	_, err = io.Copy(buf, f)
+	_, err = buf.ReadFrom(f)
```

Comment:
ReadFrom reads data until EOF and appends it to the buffer.

We used io.Copy here because it copies the data in small chunks to keep the memory usage low.

Reply:
Yes and not but you still read everything into `buf`, so memory would be the same. You could even grow the buffer to the size of the file ... so io.Copy has NO benefit here.